monitor/services/src/test/java/org/datacleaner/monitor/configuration/ComponentStoreImplTest.java

```diff
@@ -47,7 +47,7 @@ private Repository getRepositoryMock() {
 
     private RepositoryFolder getRepositoryFolderMock() {
         RepositoryFolder repositoryFolder = createNiceMock(RepositoryFolder.class);
-        expect(repositoryFolder.getFolder(ComponentStoreImpl.FOLDER_NAME)).andReturn(getComponentsFolderMock()).anyTimes();
+        expect(repositoryFolder.getOrCreateFolder(ComponentStoreImpl.FOLDER_NAME)).andReturn(getComponentsFolderMock()).anyTimes();
```

Comment:
Hmmmm, I'm not sure I agree here: If it calls it more than once, something weird is going on. Catching when code does weird stuff (even if it seems harmless at first) is a great may to discover bugs. Using anyTimes() right away is actively hiding bugs. Sure, if it is discovered that it does it more than once for a good reason, by all means add anyTimes() (or preferably times(x) if you have a good idea what x is, and x wont be easily affected by irrelevant code changes), but it's a terrible default.

(And expected execution count is certainly not just specifying the mock's behaviour, or it would only have to specified when you have differing outputs for every call. That's also why most mocking frameworks has a verification step after execution, to make sure methods has been called the minimum amount of times).

But yeah, it's old code, so not really related. It just seemed curious to me.